simplic-user/Simplic.User.UI/simplic-repository/Localization/usermanagment.de-DE.json

```diff
@@ -0,0 +1,27 @@
+﻿{
+  "usermanagment_add_user_label": "Benutzer hinzufügen",
```

Comment:
Framework localizations will not be placed in the database. They should be located under /repos/simplic-framework/dependencies/localization/global...

Reply:
Done